lib/db.js

```diff
@@ -435,7 +437,7 @@ module.exports = function (
             return bufferize({
               id: item.id,
               sessionToken: item.sessionTokenId,
-              lastAccessTime: item.lastAccessTime,
+              lastAccessTime: marshallLastAccessTime(item.lastAccessTime, uid, item.email),
```

Comment:
Fwiw, this works correctly even without `item.email` being returned from the db (it was added in https://github.com/mozilla/fxa-auth-db-mysql/pull/170). By "correctly", I mean the `enabledEmailAddresses` setting just functions as a nop.